mono/metadata/jit-icall-reg.h

```diff
@@ -339,7 +339,13 @@ MONO_JIT_ICALL (ves_icall_thread_finish_async_abort) \
 	\
 MONO_JIT_ICALL (count) \
 
-typedef enum MonoJitICallId {
+// Widen to gsize for use in MonoJumpInfo union.
+#ifdef __cplusplusplus
```

Comment:
Is the C+++ intentional?

Reply:
Yes.

Reply:
Oops just C++, not C+++, fixed, thank you.